From 5d67cbcd5ac04a5744a8b71b9468d81c8f8a378c Mon Sep 17 00:00:00 2001
From: nibon7 <nibon7@163.com>
Date: Fri, 25 Mar 2022 23:49:16 +0800
Subject: [PATCH] disable lto

CI build fails due to out of memory when lto is enabled.
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index e214da1e..bab47751 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -90,7 +90,7 @@ dataframe = ["nu-command/dataframe"]
 [profile.release]
 opt-level = "s" # Optimize for size
 strip = "debuginfo"
-lto = "thin"
+#lto = "thin"
 
 # build with `cargo build --profile profiling`
 # to analyze performance with tooling like linux perf
-- 
2.35.1

